src/sentry/models/grouphistory.py

```diff
@@ -159,6 +164,7 @@ class GroupHistory(Model):
             (GroupHistoryStatus.SET_RESOLVED_IN_COMMIT, _("Resolved in Commit")),
             (GroupHistoryStatus.SET_RESOLVED_IN_PULL_REQUEST, _("Resolved in Pull Request")),
             (GroupHistoryStatus.ESCALATING, _("Escalating")),
+            (GroupHistoryStatus.ONGOING, _("Ongoing")),
```

Comment:
Could you please also change the default to be `GroupHistoryStatus.UNRESOLVED` instead of `0`?